### PR TITLE
Implementation of new amp-ad network: medyanet

### PR DESCRIPTION
**Pull request by mozdenn**
Added new amp-ad adaptor: medyanet

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -374,6 +374,13 @@ anymatch@^1.3.0:
     micromatch "^2.1.5"
     normalize-path "^2.0.0"
 
+anymatch@^2.0.0:
```
Comment (lannka) [inline yarn.lock]: please revert this file

File: 3p/integration.js [hunk 2]
```diff
@@ -412,6 +414,7 @@ const defaultAllowedTypesInCustomFrame = [
   'doubleclick',
   'yieldbot',
   '_ping_',
+  'medyanet',
```
Comment (lannka) [inline 3p/integration.js]: this list exists for legacy reason. you don't need to change it

File: yarn.lock (hunk 1 above)
Comment (mozdenn) [reply yarn.lock]: Blank ads issue is fixed now. You should see ads for medyanet. Also I reverted yarn.lock file.

File: 3p/integration.js (hunk 2 above)
Comment (mozdenn) [reply 3p/integration.js]: I removed this line.

File: yarn.lock (hunk 1 above)
Comment (lannka) [reply yarn.lock]: did you push? I still see the yarn.lock in the PR

File: package.json [hunk 3]
```diff
@@ -42,7 +42,7 @@
   "devDependencies": {
     "acorn-globals": "4.1.0",
     "ajv": "6.1.1",
-    "ajv-keywords": "3.1.0",
+    "ajv-keywords": "2.1.1",
```
Comment (lannka) [inline package.json]: revert this as well

Comment (mozdenn) [reply package.json]: They've reverted, no longer exist in the PR

**Comment by mozdenn** [comment]
After I revert package.json and yarn.lock file, percy/amphtml test is errored. But I can not see reasons of it. How can I pass this step?

**Comment by lannka** [comment]
@rsimha-amp can you advice what was the cause for the percy failure?

**Comment by rsimha-amp** [comment]
All the diffs are due to the fact that several new tests have been added recently, and your branch doesn't contain the test pages. How recently did you sync your branch to `upstream/master`?

If you want to see the diffs, fill out the form linked from the [AMP dev guide](https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-quick.md#push-your-branch--create-a-pull-request) (search for "Percy").

**Comment by mozdenn** [comment]
Althouh I pulled changes from upstream/master and merge to my branch with it, the percy test failed again. I will try on another PR with a clear with a branch that is up to date upstream/master.

**Comment by rsimha-amp** [comment]
@mozdenn if you look at the changes in this PR, there are more than 1700 files. This means your `original/master` is not synced with `upstream/master`.

**Comment by rsimha-amp** [comment]
Here is how I do it: 
```
git checkout master
git fetch upstream master
git rebase upstream/master
git checkout <branch name>
git rebase master
```

**Comment by mozdenn** [comment]
Thanks a lot! @rsimha-amp and @lannka   myorigin/master's synced with upstream/master. So all tests except cla/google are passed successfully. Only cla/google status is still "waiting for status to be reported"

**Comment by rsimha-amp** [comment]
The CLA check failed because you had initially pushed commits from several other authors. See https://github.com/ampproject/amphtml/issues/13222#issuecomment-364361538

This cannot be undone, so we'll have to have someone override the CLA check failure.

**Comment by mozdenn** [comment]
Is there anything that I have to do for the PR to proceed?

**Comment by rsimha** [comment]
I can merge this for you after @lannka reviews and approves.

**Pull request merged**